### PR TITLE
tinyiiod: Handle buffers count.

### PR DESCRIPTION
**Pull request by teoperisanu**
Command: SET < device > BUFFERS_COUNT < count >.
Add function for setting the number of kernel buffers.

### CODE REVIEW

File: tinyiiod.c [hunk 1]
```diff
@@ -284,3 +284,14 @@ int32_t tinyiiod_set_timeout(struct tinyiiod *iiod, uint32_t timeout)
 
 	return ret;
 }
+
+int32_t tinyiiod_set_buffers_count(struct tinyiiod *iiod, const char *device, uint32_t buffers_count)
+{
+	int32_t ret = 0;
+
+	if (iiod->ops->set_timeout)
```
Comment (mchindri) [inline tinyiiod.c]: iiod->ops->set_buffers_count

**Comment by mchindri** [comment]
Where is this new feature documented?
I only know about this: https://wiki.analog.com/resources/tools-software/linux-software/libiio_internals#protocol 
Can you update it too?

File: tinyiiod.c (hunk 1 above)
Comment (teoperisanu) [reply tinyiiod.c]: Modified.

**Comment by teoperisanu** [comment]
> Where is this new feature documented?
> I only know about this: https://wiki.analog.com/resources/tools-software/linux-software/libiio_internals#protocol
> Can you update it too?

Yes, I will update the wiki page.
In libiio is a brief documentation (https://github.com/analogdevicesinc/libiio/blob/master/iiod/parser.y#L141).

**Review by mchindri — APPROVED** [review]

**Comment by teoperisanu** [comment]
v2: Add the device id as parameter of set_buffers_count function
https://github.com/analogdevicesinc/libtinyiiod/pull/35/files#diff-917893152a307d12a4fc74e19af9c59d330b67ec24cb894b0054c7680afc6094R69

**Pull request merged**